Resources/Textures/_DV/Clothing/Head/Hats/admin_hat.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "uhhhh..... free?",
```

Comment:
bruh say who made it on what platform
e.g. `Created by Urist (github) for Delta-V`